abi/erc20.go

```diff
@@ -1,5 +1,56 @@
-package abis
+package contracts
```

Comment:
Can the directory be renamed to match?

Reply:
we already have contracts, renamed to assets

Reply:
If contracts is a better fit, then the existing dir could change to `examples` or something